next.config.js

```diff
@@ -0,0 +1,24 @@
+const path = require('path')
+const { execSync } = require('child_process')
+const withCSS = require('@zeit/next-css')
+const withImages = require('next-images')
+const { version } = require('./package.json')
+
+const COMMIT_SHA =
+  process.env.COMMIT_SHA || execSync("git log --pretty=format:'%h' -n 1")
+
+const BUILD = process.env.BUILD || `${version}-${COMMIT_SHA.slice(0, 7)}`
+
+module.exports = withCSS(
+  withImages({
+    webpack(config, options) {
```

Comment:
Is webpack an injected global from Next into its config file?

Reply:
No it’s a method declaration here :-)

Equivalent to `webpack: function webpack(config, options) {`